src/nrc/conf/includes/base.py

```diff
@@ -132,10 +132,13 @@
 # Django setup configuration
 #
 SETUP_CONFIGURATION_STEPS = [
-    "nrc.config.site.SiteConfigurationStep",
-    "nrc.config.authorization.AuthorizationStep",
-    "nrc.config.authorization.OpenZaakAuthStep",
-    "nrc.config.notification_retry.NotificationRetryConfigurationStep",
+    "zgw_consumers.contrib.setup_configuration.steps.ServiceConfigurationStep",
+    "vng_api_common.contrib.setup_configuration.steps.JWTSecretsConfigurationStep",
+    "nrc.setup_configuration.authorization.AuthorizationStep",
+    # TODO these need to be implemented later
+    # "notifications_api_common.contrib.setup_configuration.steps.NotificationConfigurationStep",
+    # TODO this should be moved to `django-setup-configuration`
+    # "nrc.config.site.SiteConfigurationStep",
```

Comment:
This should either be uncomment or removed (with it's configuration step and model)

Reply:
I've removed them and added separate issues for them